perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/FacilitiesManagerBlImpl.java

```diff
@@ -340,6 +340,28 @@ public void deleteFacility(PerunSession sess, Facility facility, Boolean force)
 			}
 		}
 
+		//remove admins of this facility
+		List<Group> adminGroups = getFacilitiesManagerImpl().getAdminGroups(sess, facility);
+		for (Group adminGroup : adminGroups) {
+			try {
+				AuthzResolverBlImpl.unsetRole(sess, adminGroup, facility, Role.FACILITYADMIN);
+			} catch (GroupNotAdminException e) {
+				log.warn("When trying to unsetRole FacilityAdmin for group with id: " + adminGroup.getId() + ", in the facility with id: " + facility.getId() + " , this exception was thrown: " + e.toString());
```

Comment:
You should use format like this:
`log.warn("When trying to unsetRole FacilityAdmin for group {} in the facility {} the exception was thrown {}", adminGroup, facility, e.ToString());`
for any logging in code.

Reply:
Just a note: exception object can be passed as it is, there is no need to convert it to string using ``e.toString()``, logging framework will do it by itself.